Engine/source/console/debugOutputConsumer.h

```diff
@@ -27,8 +27,8 @@
 
 //#define TORQUE_LOCBUILD
 
-#if !defined(TORQUE_DEBUG) && defined(TORQUE_OS_XENON) && !defined(TORQUE_LOCBUILD)
-#define DISABLE_DEBUG_SPEW
+#if !defined(TORQUE_DEBUG) && !defined(TORQUE_LOCBUILD)
+   #define DISABLE_DEBUG_SPEW
```

Comment:
Interested to know whether disabling debug spew is necessary anymore since old condition required Xenon to be defined/enabled.